hub2vu/rag_chatbot.ipynb

```
@@ -0,0 +1,252 @@
+    "import os\n",
+    "import import_ipynb\n",
+    "from openai import OpenAI\n",
+    "from build_vector_db import get_embedding\n",
+    "from chromadb import Client\n",
+    "import chromadb\n",
+    "from chromadb.config import Settings\n",
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "load_dotenv()\n",
+    "dbclient = chromadb.PersistentClient(path=\"./chroma_db\")\n",
+    "collection = dbclient.get_or_create_collection(\"rag_collection\")"
```

Ruff (0.13.1)

Unused function argument: `top_k`

(ARG001)